python/upt-cpan/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                upt-cpan
+version             0.4
+revision            0
+
+maintainers         nomaintainer
+description         CPAN frontend for upt
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://framagit.org/upt/upt-cpan
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+license             3-clause BSD License
+checksums           sha256  47ebc37bf4510867858eec52e2d23221bc5a0f061a057b41892cb96b8e3def14 \
+                    rmd160  418532c5abb388d4ddcb050610944266ba1f456c \
+                    size    5091
+                                
+python.versions     37
+
+
+depends_build-append \
+                    port:py${python.version}-setuptools
+
+depends_lib-append \
+                    port:py${python.version}-requests \
+                    port:upt
```

Comment:
Is upt really needed already at the time of installing this port? (I didn't test, I'm just asking.)

In case you can install this port without having `upt` installed upfront, this would be part of `depends_run-append` instead.

Reply:
see my comment above. I think the differentiation between ```depends_lib``` and ```depends_run``` is not made in many Python ports I've looked at. That's not to say that we should pick the most correct declaration of dependencies..